lib/record-serialization.js

```diff
@@ -11,25 +11,25 @@ module.exports = jsrs;
 // one of our priorities to optimize it.
 const USE_NATIVE_SERIALIZER = false;
 
-let jsrsNative;
+let jstpNative;
 
 try {
-  jsrsNative = require('../build/Release/jsrs');
+  jstpNative = require('../build/Release/jstp');
```

Comment:
To avoid enclosure `try`, better create safe require
```js
const safeRequire = name => {
  try {
    return require(name);
  } catch (e) {
    return e;
  }
};
```